workspaces/jupyterlab/Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM jupyter/datascience-notebook
 COPY jupyter_notebook_config.py /etc/jupyter/
+ENV JUPYTER_RUNTIME_DIR /tmp
```

Comment:
(this was another suggestion on some Jupyter github issue thread, to isolate things slightly more)

```suggestion
ENV JUPYTER_RUNTIME_DIR /tmp/runtime
```

Reply:
works on my end